app/build.gradle.kts

```diff
@@ -86,21 +86,12 @@ android {
         getByName("debug") {
             isMinifyEnabled = false
 
-            versionNameSuffix = "-dev"
-            applicationIdSuffix = ".dev"
+            applicationIdSuffix = ".debug"
```

Comment:
I'm not sure how this suffix issue will go, before we used it to differentiate build types 🤔

Reply:
Could be `dev`, `debug` etc. It's for app id to be able to install `release` for everyday use and `debug` for development/testing on the same device.

Reply:
Perfect. Now the `versionName` will always have the suffix "-dev" until the version is finalized. So instead of bumping the version in the release branch, we'll bump it as soon as we finish the previous one and start the new one, using `VersionConfig.Type.DEV` to indicate that it's not ready. When it's ready, just switch to `VersionConfig.Type.RELEASE`. This way, the Git history remains more concise. You could even create `VersionConfig.Type.PRE_RELEASE`, `VersionConfig.Type.BETA`, `VersionConfig.Type.ALPHA`, etc.

For example, right now in the develop branch, the configured version is v0.1.2, but that's already been released. This is confusing. Bumping the version only when it's finalized makes more sense. Bumping it as soon as development starts is more concise, but it might not be as clear in the history where the previous one left off.

Reply:
The fact that it's a concrete type also makes the project's pattern clear.

Reply:
I updated the description.